### PR TITLE
languagetool: add service to run languagetool-server

### PR DESCRIPTION
**Pull request by moabits**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Adds a service block to the formula to run the languagetool-server.
@languagetool-org

### CODE REVIEW

File: Formula/languagetool.rb [hunk 1]
```diff
@@ -43,6 +43,36 @@ def install
     EOS
   end
 
+  plist_options manual: "#{HOMEBREW_PREFIX}/bin/languagetool-server --port 8081 --allow-origin"
+
+  def plist
```
Comment (carlocab) [inline Formula/languagetool.rb:48]: Please make this a `service` block instead of a `plist`.

Comment (moabits) [reply Formula/languagetool.rb]: Done! Can you update the [documentation](https://docs.brew.sh/Formula-Cookbook#launchd-plist-files) sometime to point towards the preferred format?

**Review by carlocab — APPROVED** [review]
Thanks!

File: Formula/languagetool.rb (hunk 1 above)
Comment (carlocab) [reply Formula/languagetool.rb]: The `service` block API is still a work in progress, so we'll want to wait until it's closer to stable when we add it to the documentation.

Comment (carlocab) [reply Formula/languagetool.rb]: The `service` block API is still a work in progress, so we'll want to wait until it's closer to stable when we add it to the documentation.

**Pull request merged**